pages/index.js

```diff
@@ -1,30 +1,79 @@
-import React from "react";
+import React, { useState } from 'react';
+import styled from 'styled-components';
 
-import TodoInput from "../components/todo-input";
-import TodoList from "../components/todo-list";
-
-import styled from "styled-components";
+import TodoInput from '../components/todo-input';
+import TodoList from '../components/todo-list';
 
 export default function Home() {
-  return (
-    <Wrapper>
-      <Contents>
-        <TodoInput />
-        <TodoList />
-      </Contents>
-    </Wrapper>
-  );
+	const [date, setDate] = useState();
+	const [todo, setTodo] = useState();
+	const [todos, setTodos] = useState([]);
+	const [id, setID] = useState(0);
+
+	const changeTodo = (e) => {
+		setTodo(e.target.value);
+	};
+
+	const changeDate = (e) => {
+		setDate(e.target.value);
+	};
+
+	const removeTodo = (id) => {
+		const newTodos = [...todos];
+		setTodos(newTodos.filter((info) => info.id !== id));
+	};
+	const submitTodo = (e) => {
+		e.preventDefault();
+		if (todo === '' || date === '') {
+			alert('모든 항목을 입력해주세요!');
+			return false;
+		} else if (date.length != 8) {
+			alert('날짜를 올바른 형식으로 입력해주세요!');
+			return false;
+		} else alert('입력되었습니다!');
+		setTodos(todos.concat({ id: id, date: date, todo: todo }));
+		setTodo('');
+		setDate('');
+		setID(id + 1);
+	};
+
+	return (
+		<Wrapper>
+			<Subject>리액트-투두</Subject>
+			<Contents>
+				<TodoInput
+					date={date}
+					todo={todo}
+					onChangeDate={changeDate}
+					onChangeTodo={changeTodo}
+					onSubmit={submitTodo}
+				/>
+				<Space />
+				<TodoList data={todos} onRemove={removeTodo} />
+			</Contents>
+		</Wrapper>
+	);
 }
+const Space = styled.div`
+	flex: 0.3;
+`;
 
 const Wrapper = styled.div`
-  height: 100vh;
+	height: 100vh;
+	font-size: 20px;
+	background-color: rgb(155, 197, 195);
+	padding: 4rem 5rem;
+`;
+const Subject = styled.div`
+	font-size: 3rem;
+	font-weight: 600;
+	padding: 0px;
+	margin: 0px 0px 3rem;
+	color: white;
```

Comment:
```suggestion
const Title = styled.h1`
	font-size: 3rem;
	font-weight: 600;
	margin-bottom: 3rem;
	color: white;
```
1. 제목을 나타내는 태그이니까 네이밍을 ```Title```로 하면 좋을 것 같아요.
2. 제목을 적는 태그라서 ```div```보단 ```h1```을 사용하면 좋습니다.
3. ```padding: 0px``` 코드는 꼭 필요해 보이진 않네요!
4. 아래에만 margin을 주고 싶다면 ```margin-bottom: 3rem```과 같이 써주시면 돼용